lib/cloudant.js

```diff
@@ -292,6 +293,25 @@ function findRecursive(mo, query, docs, include, cb) {
   mo.db.find(query, function(err, rst) {
     debug('Cloudant.prototype.all (findRecursive) results: %j %j', err, rst);
     if (err) return cb(err);
+
+    // sort back the ids in "numerical order" if they are numeric
+    query.sort.forEach(function(queries) {
```

Comment:
Why are we sorting the list for them? What if they don't want the list sorted? Now they have no choice!